libraries/AP_Relay/AP_Relay_Params.h

```diff
@@ -29,9 +29,45 @@ class AP_Relay_Params {
         BRUSHED_REVERSE_3 = 7,
         BRUSHED_REVERSE_4 = 8,
         ICE_STARTER = 9,
+        DroneCAN_HARDPOINT_0  = 10,
+        DroneCAN_HARDPOINT_1  = 11,
+        DroneCAN_HARDPOINT_2  = 12,
+        DroneCAN_HARDPOINT_3  = 13,
+        DroneCAN_HARDPOINT_4  = 14,
+        DroneCAN_HARDPOINT_5  = 15,
+        DroneCAN_HARDPOINT_6  = 16,
+        DroneCAN_HARDPOINT_7  = 17,
+        DroneCAN_HARDPOINT_8  = 18,
+        DroneCAN_HARDPOINT_9  = 19,
+        DroneCAN_HARDPOINT_10 = 20,
+        DroneCAN_HARDPOINT_11 = 21,
+        DroneCAN_HARDPOINT_12 = 22,
+        DroneCAN_HARDPOINT_13 = 23,
+        DroneCAN_HARDPOINT_14 = 24,
+        DroneCAN_HARDPOINT_15 = 25,
         NUM_FUNCTIONS // must be the last entry
     };
 
+    // Pins that do not go via GPIO
+    enum class VIRTUAL_PINS {
+        DroneCAN_0  = 1000,
+        DroneCAN_1  = 1001,
+        DroneCAN_2  = 1002,
+        DroneCAN_3  = 1003,
+        DroneCAN_4  = 1004,
```

Comment:
go for 6 for now, but reserve up to 16

Reply:
I have an existing need for more then 6 already, and was going to PR expanding the number of relays relatively soon, I'd much rather have all 16 now.

Reply:
We now support 16 relays, so I have left at 16 virtual pins here.